CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 
 ### Added
 - #1060 - New tree activation MCP utility
+- #New workflow process step "Set Image Orientation"
```

Comment:
Can you update this file to reference the PR number, 2434

Reply:
Done - also moved to "Unreleased changes" which I think is correct place.